ckanext/stadtzhtheme/fanstatic/stadtzhtheme.css

```diff
@@ -385,3 +385,8 @@ ul.resource-list,
         float: left;
     }
 }
+
+/*  */
```

Comment:
Why is there an empty comment?

Reply:
Sorry, I forgot to fill that in. Fixed now :)